sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -169,7 +169,7 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
   }
 
   private def checkPushedLimit(df: DataFrame, limit: Option[Int] = None,
-      sortValues: Seq[SortValue] = Nil): Unit = {
+      sortValues: Seq[SortValue] = Nil, sortRemoved: Boolean = false): Unit = {
```

Comment:
I think the default value of `sortRemoved` should be true. It's more interesting to see which tests can't remove the sort while top N is pushed down.